### PR TITLE
feat(cf): add makefile target to diff UPS vars

### PR DESCRIPTION
**Pull request by dagoof**

### CODE REVIEW

**Comment by andrewoh531** [comment]
Is this the final solution to our secrets management or an intermediate tool?

**Comment by jonathaningram** [comment]
> Is this the final solution to our secrets management or an intermediate tool?

@dagoof? I see this as intermediate.

File: Makefile [hunk 1]
```diff
@@ -80,6 +82,13 @@ $(SVC_APPLIED): apply-service-%: ci/ups/$(CLD_HOST)/%.json
 $(SVC_CREATED): create-service-%:
 	$(MAKE) apply-service-$* APPLY_ACTION=create
 
+$(SVC_DIFFED): SHELL = /bin/bash
+$(SVC_DIFFED): diff-service-%:
+	@diff \
```
Comment (jonathaningram) [inline Makefile]: @dagoof could we use `sdiff` or `diff -s`? More useful output IMO.

Comment (dagoof) [reply Makefile]: tried sdiff, why is it more useful here? instead of listing the changes mine is now echoing out both identical files

**Pull request closed without merging**